### PR TITLE
Fixed incorrect multiplier being used for payout currency in exchange data

### PR DESCRIPTION
**Pull request by jacobburrell**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [x ] Tested on iOS emulator
- [ ] n/a

### CODE REVIEW

**Comment by jacobburrell** [comment]
@thehobbit85 I believe it would be best to fix the prop " passed in.

Currently, every coin returns mostly a destinationDenomination object with blank values except for "multiplier" which always has a value of 1.

destinationDenomination.multiplier should be what I'm looking up via "payoutMultiplier" via settings object I pulled from TransactionDetailsScene's connector.

File: src/components/scenes/TransactionDetailsScene.js [hunk 1]
```diff
@@ -251,16 +252,22 @@ export class TransactionDetailsComponent extends React.Component<Props, State> {
     ))
   }
 
-  renderExchangeData = (symbolString: string) => {
-    const { destinationDenomination, destinationWallet, edgeTransaction, guiWallet, walletDefaultDenomProps, theme } = this.props
+
+  renderExchangeData = () => {
```
Comment (thehobbit85) [inline src/components/scenes/TransactionDetailsScene.js:256]: By removing "(symbolString: string)" it breaks the code.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

File: src/components/scenes/TransactionDetailsScene.js (hunk 1 above)
Comment (jacobburrell) [reply src/components/scenes/TransactionDetailsScene.js:255]: Added back in.

Comment (paullinator) [reply src/components/scenes/TransactionDetailsScene.js:255]: What is this change tested at all? You created a syntax error which could not possibly run

**Review by paullinator — CHANGES_REQUESTED** [review]
See comments. As well fix the conflict by rebasing your changes

File: src/components/scenes/TransactionDetailsScene.js (hunk 1 above)
Comment (jacobburrell) [reply src/components/scenes/TransactionDetailsScene.js:255]: Tested and rebased ontop of Develop. 
image

**Comment by paullinator** [comment]
This PR as it stands cannot be merged! There are merge commits that go from develop into this feature branch which is not allowed per our workflow

**Pull request closed without merging**